### ISSUE TITLE
Add Label Conversion to Nifti Conversion

### ISSUE THREAD

**Issue by leowe** [scalableminds/webknossos-cuber#181]
- currently segmentation data is converted the same way as gray scale data 
- could use https://scikit-learn.org/stable/modules/generated/sklearn.preprocessing.LabelEncoder.html
- use behavior that converts sparse labels to dense labels beginning at 0

**Comment by hotzenklotz** [comment]
@leowe @philippotto Is this still an issue?

**Comment by leowe** [comment]
Yes this is still an issue. I think to be actually useful we also need to add the option to use this conversion in a color layer, since we can't show multiple segmentation layers in webknossos

**Comment by leowe** [comment]
This is already resolved.
